packages/core/resolve-client/src/client.ts

```diff
@@ -71,11 +71,9 @@ export const command = (
   asyncExec()
```

Comment:
Expression like `Promise.resolve().then(asyncExec)` will be better since should handle potential synchronous errors throwing